help_to_heat/portal/download_views.py

```diff
@@ -96,6 +121,23 @@ def add_extra_row_data(referral):
     return row
 
 
+def match_rows_for_feedback(feedback):
+    created_at = feedback.created_at.astimezone(london_tz)
+    page_name = feedback.page_name
+    row = dict(feedback.data)
+    row = {
+        **row,
+        "useful_for_learning": row["how-much"],
```

Comment:
Have renamed the columns so it's more obvious which question in the feedback each column corresponds to